src/pages/login.tsx

```diff
@@ -6,7 +6,7 @@ import stringToBool from '../utils/stringToBool';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const Z = require('zetkin');
 
-export const getServerSideProps : GetServerSideProps = async () => {
+export const getServerSideProps : GetServerSideProps = async (context) => {
     const z = Z.construct({
```

Comment:
can this use the little z from context? or does it not work to do that here?

Reply:
The `z` property doesn't exist on `context` by default, but is added to it by the `scaffold()` function, which isn't used here.